app/controllers/demo_controller.rb

```diff
@@ -1,4 +1,22 @@
 class DemoController < ApplicationController
+  def reset
+    set_current_user({beverage: 0})
+    session[:percentage] = 0
+    Flipper.disable_percentage_of_actors(:slow_roll)
+    Flipper.disable_percentage_of_time(:slow_roll)
+    Flipper.disable(:demo)
+    Flipper.disable(:demo_actor)
+    Flipper.disable(:demo_group)
+    Flipper.disable(:slow_roll)
```

Comment:
Maybe `reset_session` and `Flipper.features.each(&:disable)` to make this more future proof?